src/shared/telemetry/exemptMetrics.ts

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * TODO: Fix calls to these metrics and remove this file.
+ * NOTE: Do NOT add additional metrics here, your PR will be rejected. All new metrics should pass validation.
+ *
+ * Checks were added to all emitted telemetry metrics to ensure they contain the correct properties.
+ * This check does not exist outside test/automated runtimes because the error is not useful
+ * for the user and doesn't break telemetry.
+ *
+ * These are metrics that fail validation and are emitted by code that is covered in test cases.
+ * This allowlist exists to fix these incrementally and not have CI fail in the meantime.
+ */
+const validationExemptMetrics: Set<string> = new Set([
+    'amazonq_runCommand',
+    'amazonq_isReviewedChanges',
+    'apigateway_copyUrl',
+    'aws_loadCredentials',
+    'aws_validateCredentials',
+    'cloudwatchlogs_download',
```

Comment:
amazonq open/close chat doesnt have result anymore. do we need to include those metrics in this list?

Reply:
those metrics didn't fail for me when i was running tests, so it looks like the code that runs those is untested. We can only enforce for metrics that run in test cases at the moment- don't want to throw errors to the user during runtime.